tests/unit/synapseclient/unit_test_client.py

```diff
@@ -2999,15 +2999,25 @@ def test_is_certified__no_quiz_results(syn):
     response.status_code = 404
     with patch.object(syn, "getUserProfile",
                       return_value={"ownerId": "foobar"}) as patch_get_user,\
-         patch.object(syn,
-                      "_get_certified_passing_record",
-                      side_effect=SynapseHTTPError(response=response)) as patch_get_cert:
+        patch.object(syn,
+                     "_get_certified_passing_record",
+                     side_effect=SynapseHTTPError(response=response)) as patch_get_cert:
         is_certified = syn.is_certified("test")
     patch_get_user.assert_called_once_with("test")
     patch_get_cert.assert_called_once_with("foobar")
     assert is_certified is False
 
 
+def test_is_synapse_id(syn):
+    # Invalid IDs
+    assert syn.is_synapse_id("test") is False
+    assert syn.is_synapse_id("123") is False
```

Comment:
Nit (Nitpicky): Is there a reason you don't do:
```
assert not syn.is_synapse_id("test")
assert not syn.is_synapse_id("123")
```

Reply:
lol because it was Home Week and I forgot all proper coding rules...